Formula/wxmac-stable.rb

```diff
@@ -0,0 +1,89 @@
+class WxmacStable < Formula
```

Comment:
Do we really want to name it like this and not make it the versioned formula (like proposed in the PR title "wxmac@3.0")? I don't like the name `wxmac-stable` because it implies that `wxmac` is unstable, which is not true

Reply:
I'm open to an alternative name, but I think we don't want `wxmac@3.0` because we don't really care about the 3.0.x branch -- we just want what they call their "stable" release. See also discussion at https://github.com/Homebrew/homebrew-core/pull/79798#discussion_r656428961.

I contemplated `wxmac-lts` but that seems confusing for users since it doesn't actually correspond to any language that upstream uses. The term that upstream uses is "stable" -- it just so happens that upstream's use of "stable" isn't quite the same as the way we use "stable".